File: Assignment 11/Program5.c
#include<stdio.h>

int Count(int iNo)
{
 int iDigit=0;
    int icnt=0;
    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if (iDigit < 6)
        {
            icnt++;
        }
        iNo = iNo / 10;
    }
    return icnt; 
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);

    iRet = Count(iValue);

printf("%d",iRet);
    return 0;
}

File: Assignment No 8/Program5.java
//Problem Statement : Write a Program which return difference between Even Factorial and Odd Factorialof given number.
//Input : 5
//Output : -7
//Input : -5
//Output : -7
//Input : 10
//Output : 2895
import java.io.*;

class Program5
{
	public static int FactorialDiff(int iValue)
	{
		if(iValue < 0)
		{
			iValue=-iValue;
		}
		int iEven=1,iOdd=1;
		for(int i=iValue;i>=1;i--)
		{
			if(i%2==0)
			{
				iEven=iEven*i;
			}
			else
			{
				iOdd=iOdd*i;
			}
		}
		return(iEven-iOdd);
	}
	public static void main(String args[])throws Exception
	{
		BufferedReader bobj=new BufferedReader(new InputStreamReader(System.in));

		int iNo=0,iRet=0;

		System.out.println("Enter the number");
		iNo=Integer.parseInt(bobj.readLine());

		iRet=FactorialDiff(iNo);

		System.out.println("Factorial difference is "+iRet);
	}
}